example/src/ListWithHeader/ListWithHeader.tsx

```diff
@@ -0,0 +1,144 @@
+import React, { useEffect } from 'react';
+import {
+  Platform,
+  ScrollViewProps,
+  SectionList,
+  SectionListProps,
+} from 'react-native';
+import Animated, {
+  SharedValue,
+  useAnimatedRef,
+  useScrollViewOffset,
+  useAnimatedReaction,
+  useSharedValue,
+  useAnimatedProps,
+  useAnimatedStyle,
+  runOnJS,
+  withSpring,
+} from 'react-native-reanimated';
+import Header, { HEADER_HEIGHT } from './Header';
+import {
+  Gesture,
+  GestureDetector,
+  GestureType,
+} from 'react-native-gesture-handler';
+
+const IS_ANDROID = Platform.OS === 'android';
+
+export function ListWithHeader<ItemT, SectionT>(
+  props: SectionListProps<ItemT, SectionT>
+) {
+  const scrollOffset = useSharedValue(0);
+  const scrollEnabled = useSharedValue(true);
+  const androidDragDist = useSharedValue(0);
+
+  function enableScroll() {
+    setTimeout(() => {
+      scrollEnabled.value = true;
+    });
```

Comment:
Why do we need this `setTimeout`?

Reply:
Well, we don't anymore 😄. Thanks for pointing it out.